sdk/ml/azure-ai-ml/azure/ai/ml/_utils/_logger_utils.py

```diff
@@ -20,11 +20,11 @@ def initialize_logger_info(module_logger: logging.Logger, terminator="\n") -> No
 
 class OpsLogger:
     def __init__(self, name: str):
-        # self.logger: logging.Logger = logging.getLogger(AML_INTERNAL_LOGGER_NAMESPACE + name)
-        # self.logger.propagate = False
+        self.logger: logging.Logger = logging.getLogger(AML_INTERNAL_LOGGER_NAMESPACE + name)
```

Comment:
Minor: the variable name `logger` is too generic and can cause confusion. Consider a more specific name.

Reply:
Fair. I renamed to "package_logger" to differentiate from "module_logger"